examples/nlp/language_modeling/megatron_t5_pretraining.py

```diff
@@ -45,7 +46,7 @@ def main(cfg) -> None:
     if cfg.get('cluster_type', None) == 'BCP':
         plugins.append(TorchElasticEnvironment())
 
-    trainer = Trainer(plugins=plugins, **cfg.trainer)
+    trainer = Trainer(plugins=plugins, **cfg.trainer, callbacks=[ModelSummary(max_depth=3)])
```

Comment:
A question: do we want to limit max_depth to 3? Does it capture enough of the encoder-decoder structure?

Reply:
PTL butchers the ModelSummary (misses certain lines or has incomplete/truncated lines) both on stdout and in the logfile for larger depths :(